contrib/mixin/mixin.libsonnet

```diff
@@ -238,6 +238,20 @@
               summary: 'etcd cluster database growing very fast.',
             },
           },
+          {
+            alert: 'etcdBackendHighFragmentationRatio',
+            expr: |||
+              (etcd_mvcc_db_total_size_in_use_in_bytes/etcd_mvcc_db_total_size_in_bytes)*100 < 50
+            ||| % $._config,
+            'for': '10m',
+            labels: {
+              severity: 'warning',
+            },
+            annotations: {
+              description: 'etcd cluster "{{ $labels.%s }}": database size in use on instance {{ $labels.instance }} is less than half of the actual allocated disk space, please run compaction and defragmentation procedures to retrieve the unused fragmented disk space.' % $._config.clusterLabel,
```

Comment:
Just one minor comment, suggest to change "`please run compaction and defragmentation procedures`" to "`please run defragmentation procedure`". 

Please note that compaction is only related to the historical data (MVCC), but defragmentation is only related to `size_in_use / size_allocated`.

Reply:
@ahrtr done 👍🏽